Rdmp.Core/Rdmp.Core.csproj

```diff
@@ -1,382 +1,385 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <PackageId>HIC.RDMP.Plugin</PackageId>
-    <version>$(version)</version>
-    <title>HIC.RDMP.Plugin</title>
-    <authors>Health Informatics Centre, University of Dundee</authors>
-    <owners>Health Informatics Centre, University of Dundee</owners>
-    <licenseUrl>https://raw.githubusercontent.com/HicServices/RDMP/master/LICENSE</licenseUrl>
-    <projectUrl>https://github.com/HicServices/RDMP</projectUrl>
-    <iconUrl>
-      https://raw.githubusercontent.com/HicServices/RDMP/master/Application/ResearchDataManagementPlatform/Icon/main.png</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <PackageLicenseExpression>GPL-3.0-or-later</PackageLicenseExpression>
-    <description>Core package for plugin development</description>
-    <copyright>Copyright 2018-2019</copyright>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
```

Comment:
Sudden tab explosion here?

Reply:
I ran the visual studio formatter on this file, not sure why it's such a large change...

Reply:
It decided to replace all the leading spaces with tabs, so virtually every line has a "change" - probably a setting that can be adjusted somewhere.